polyglot-kotlin/src/test/resources/unit-tests/dependencies/variation-1/pom.kts

```diff
@@ -0,0 +1,49 @@
+project {
+    dependencies {
+        dependency {
+            groupId = "org.springframework.boot"
+            artifactId = "spring-boot-starter"
+            version = "2.1.2.RELEASE"
+            type = "jar"
+            scope = "compile"
+            isOptional = false
+        }
+        dependency {
+            groupId = "org.springframework.boot"
+            artifactId = "spring-boot-devtools"
+            version = "2.1.2.RELEASE"
+            scope = "runtime"
+            isOptional = true
+        }
+        dependency {
+            groupId = "org.springframework.boot"
+            artifactId = "spring-boot-configuration-processor"
+            version = "2.1.2.RELEASE"
+            scope = "provided"
+            isOptional = true
+        }
+        dependency {
+            groupId = "org.example"
+            artifactId = "custom-lib"
+            version = "1.0"
+            classifier = "jdk11"
+            scope = "system"
+            systemPath = "lib/custom-lib-1.0.jar"
+        }
+        dependency {
+            groupId = "org.springframework.boot"
+            artifactId = "spring-boot-starter-test"
+            scope = "test"
+            exclusions {
+                exclusion {
+                    groupId = "junit"
+                    artifactId = "junit"
+                }
+                exclusion {
```

Comment:
This is inconsistent with another style, where it calls "excluding"

Reply:
I agree. It just felt weird when invoking it as a function for it not to be a verb.

Reply:
Making it all into verbs is going to be a large sweeping change .. I would rather not do that at this stage.. so backing out and just sticking with exclusion for now seems good.